napari/components/_tests/test_multichannel.py

```diff
@@ -120,7 +120,14 @@ def test_multichannel(shape, kwargs):
             assert viewer.layers[i].blending == 'additive'
         for key, expectation in kwargs.items():
             # broadcast exceptions
-            if key in {'scale', 'translate', 'contrast_limits', 'metadata'}:
+            if key in {
+                'scale',
+                'translate',
+                'rotate',
+                'scale',
```

Comment:
```suggestion
                'shear',
```
?

(scale is repeated.)

Reply:
fixed